### DISCUSSION TITLE
CSS escalável

### DISCUSSION THREAD

**Discussion by guilhermejcgois** [frontendbr/forum#1052] [Discussão]
Fala pessoal, tudo certo? LinkedIn recentemente soltou um [artigo](https://engineering.linkedin.com/blog/2018/04/css-at-scale--linkedins-new-open-source-projects-take-on-stylesh) sobre CSS escalável usando Blocks e OptiCSS, alguém ai está usando? Pretende dar uma brincada, como está o sentimento da comunidade quanto a isso?

**Comment by tarciozemel** [comment]
Não conhecia ainda. Parece bem legal. Com certeza vou "dar uma brincada". :grin: 

PS: aqui nossos 50 centavos para ajudar na divulgação: http://desenvolvimentoparaweb.com/css/css-blocks-linkedin-otimizador-css/
